src/granite/query/executors/value.cr

```diff
@@ -10,15 +10,11 @@ module Granite::Query::Executor
       # db.scalar raises when a query returns 0 results, so I'm using query_one?
       # https://github.com/crystal-lang/crystal-db/blob/7d30e9f50e478cb6404d16d2ce91e639b6f9c476/src/db/statement.cr#L18
 
-      raise "No default provided" unless @default
+      raise "No default provided" if @default.nil?
```

Comment:
Is this just a personal preference change?

Reply:
No, it's required since for the `.exists?` assembler, I'm passing `false` as the default, which was causing it to always raise.